scripts/rff/historical_weather_variability/4km/preprocess_4km.py

```diff
@@ -0,0 +1,98 @@
+import csv
+import glob
+import numpy as np
+import os
+from osgeo import gdal
+import requests
+import sys
+
+SCRIPTS_DIR = os.path.dirname(os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+sys.path.append(SCRIPTS_DIR)
+from rff import util
+
+bandname_to_gdcStatVars = {
+    "std_dev": "StandardDeviation_<c_var>",
+    "skewness": "Skewness_<c_var>",
+    "kurtosis": "Kurtosis_<c_var>",
+    "hpi": "HeavyPrecipitationIndex",
+    "cdd": "ConsecutiveDryDays"
+}
+
+def get_grid_latlon(ds, x, y):
+    xmin, xres, _, ymax, _, yres = ds.GetGeoTransform()
+    return ymax + y*yres, xmin + xres*x
+
+def create_4km_grids(output_csv, sample_gtiff):
+    sample_ds = gdal.Open(sample_gtiff)
+    raster = sample_ds.GetRasterBand(1).ReadAsArray()
+    shape = raster.shape
+    headers = ['latitude', 'longitude', 'typeOf', 'dcid', 'containedInPlace']
```

Comment:
nit: typeOf is going to be the same in every row, so it can be part of the TMCF.